data/human/human missions.txt

```diff
@@ -107,7 +107,269 @@ mission "Pookie, Part 2"
+				goto next
+			label repeat
+			`You approach a receptionist to put a description of Puny on a planetwide notice board.`
+			label next
+			`	The next day, you visit the receptionist. When she sees you, she tells you that one message came in response, saying that the dog was theirs. You copy the address provided and hail down a taxi.`
+			`	The address brings you to a stark white mansion upon a cliff overlooking the ocean. A path snakes down from the entrance of the mansion to a large, gold-plated gate. You bring Puny out from the taxi and ring the doorbell next to the gate. A minute later, a servant in a prim suit approaches.`
+			choice
+				`	"I'm <first>, and I'm here to return a missing pet."`
+				`	(Wait quietly.)`
+					goto inside	
+			`	The servant says, "Ah, you're the one with the dog. Come inside, mistress has been expecting you."`
+				goto enter
+			label inside
+			`	When he spots you, the servant says, "Welcome. Come inside, mistress has been expecting you."`
+			label enter
+			`	The walk to the front of the house is arduous; you pass through several different stadium-sized gardens along the way, each dedicated to a different type of biome. Looking up at the sky, you see the remnants of clouds contort unnaturally around invisible pillars. As you follow the servant, he looks back at you repeatedly, checking that you're still following. You decide to look behind yourself, and see Puny eagerly following. She has somewhat recovered from her time on Freedom, but her body is still little more than skin and bone.`
+			`	By the time you reach the mansion, your skin is slightly sunburnt. The servant walks up to the oddly tall door and presses the doorbell before moving to the side, hands folded behind his back. The bell is answered by a woman with a monstrous hairdo, barely able to fit in the doorway. "Walters, who is this?" she says in a posh Paradise World accent.`
+			`	"It's the captain who put out a description of Pookie, Madam Pemefrey," the servant replies.`
+			`	When she hears this, Pemefrey's expression changes from annoyance to elation. "Really? Oh, I can't thank enough for this, dearie. How was it rescuing her? Did you have to barter with smugglers to track her down? Chase down kidnappers to free her? Battle with warlords for her safety?"`
+			choice
+				`	"It was an adventure and a half saving her."`
+				`	"It wasn't anything too hard."`
+					goto humble
+			`	"I'm sure it was. Now, where's my darling Pookie?"`
+				goto show
+			label humble
+			`	"Oh. Well, where's my darling Pookie?"`
```

Comment:
If this is in fact the correct dog then as Peter mentioned, she should probably be reacting to seeing her owner. But perhaps Pemefrey could pass it off as you just having trained the dog to look happy in order to catch a quick buck.

And if this is in fact Pookie then since the player already transported Pookie before then there should probably be some sort of comment about that. "Wait, Pookie? You still owe me for those urine stains in my ship."